x-pack/plugins/cloud_security_posture/server/routes/benchmark_rules/find/v1.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import semverValid from 'semver/functions/valid';
+import semverCompare from 'semver/functions/compare';
+import { NewPackagePolicy } from '@kbn/fleet-plugin/common';
+import { SavedObjectsClientContract } from '@kbn/core-saved-objects-api-server';
+import { getBenchmarkFilter } from '../../../../common/utils/helpers';
+import { CSP_BENCHMARK_RULE_SAVED_OBJECT_TYPE } from '../../../../common/constants';
+import { PACKAGE_POLICY_SAVED_OBJECT_TYPE } from '../../benchmarks/benchmarks';
+import { getBenchmarkFromPackagePolicy } from '../../../../common/utils/helpers';
+
+import type {
```

Comment:
this should explicitly use v1 here instead
think of the use case of adding a newer version

Reply:
👍 will do as part of https://github.com/elastic/kibana/pull/173066/